features/step_definitions/work_related_steps.rb

```diff
@@ -159,25 +159,40 @@
   step %{I should see "Followup by remixer" within ".afterword .children"}
 end
 
+Then "I should see the related work listed on the original work" do
+  step %{I should see "See the end of the work for other works inspired by this one"}
+  step %{I should see "Works inspired by this one:"}
+  step %{I should see "Followup by remixer"}
+end
+
 Then /^I should not see the related work listed on the original work$/ do
   step %{I should not see "See the end of the work for other works inspired by this one"}
   step %{I should not see "Works inspired by this one:"}
   step %{I should not see "Followup by remixer"}
 end
 
+Then "I should not see the inspiring parent work in the beginning notes" do
+  step %{I should not see "Inspired by Parent Work by testuser" within ".preface .notes"}
+end
+
 # then for translations
 
 Then /^a parent translated work should be seen$/ do
   step %{I should see "Work was successfully posted"}
   step %{I should find a list for associations}
-  step %{I should see "A translation of Worldbuilding by inspiration" within ".preface .notes"}
+  step %{I should see "Translation of Worldbuilding by inspiration" within ".preface .notes"}
 end
 
-Then /^I should see the translation in the beginning notes$/ do
+Then "I should see the translation in the beginning notes" do
   step %{I should see "Translation into Deutsch available:" within ".preface .notes"}
   step %{I should see "Worldbuilding Translated by translator" within ".preface .notes"}
 end
 
+Then /^I should see the translation listed on the original work$/ do
```

Comment:
This regular expression was missed when doing the conversion to cucumber expressions.